<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlcgui</class>
 <widget class="QMainWindow" name="dlcgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlcgui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="mainTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="markTab">
        <attribute name="title">
         <string>mark</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTabWidget" name="markTabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="markPathsTab">
            <attribute name="title">
             <string>paths</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="2" column="1">
                <widget class="QLineEdit" name="pathsNetworkVideoLineEdit"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="pathsLocalLineEdit"/>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="pathsImportLineEdit"/>
               </item>
               <item row="4" column="2">
                <widget class="QPushButton" name="pathsImportButton">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="pathsLocalLabel">
                 <property name="text">
                  <string>Local project folder</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pathsLocalButton">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="pathsNetworkVideoLabel">
                 <property name="text">
                  <string>Video folder</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="pathsNetworkVideoButton">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="pathsImportLabel">
                 <property name="text">
                  <string>Import project folder</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pathsImportPathsButton">
               <property name="text">
                <string>Import paths</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="markParamsTab">
            <attribute name="title">
             <string>params</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="4" column="0">
                  <widget class="QLabel" name="paramSelectionLabel">
                   <property name="text">
                    <string>Selection</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QComboBox" name="paramSelectionComboBox">
                   <item>
                    <property name="text">
                     <string>uniform</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>k-means</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="paramNameLabel">
                   <property name="text">
                    <string>Marker Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="paramDataLabel">
                   <property name="text">
                    <string>Date</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="paramNameLineEdit"/>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="paramDateLineEdit"/>
                 </item>
                 <item row="6" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLineEdit" name="paramCropMarginsXMin">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="paramCropMarginsXMax">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="paramCropMarginsYMin">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="paramCropMarginsYMax">
                     <property name="text">
                      <string>0</string>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="paramNumFrameLabel">
                   <property name="text">
                    <string>Num Frames</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="paramNumFrameLineEdit"/>
                 </item>
                 <item row="5" column="0">
                  <widget class="QCheckBox" name="paramCroppingCheckBox">
                   <property name="text">
                    <string>Cropping?</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0">
                  <widget class="QLabel" name="paramMarkingsLabel">
                   <property name="text">
                    <string>Markings</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="1">
                  <widget class="QLineEdit" name="paramMarkingsLineEdit"/>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="paramMarginsLabel">
                   <property name="text">
                    <string>Crop margins</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="paramProjectNameLabel">
                   <property name="text">
                    <string>Project name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="paramProjectNameLineEdit">
                   <property name="text">
                    <string>TrackingMyCatWhyNot14_43</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="leftMargin">
                  <number>50</number>
                 </property>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QPushButton" name="paramSampleImagesButton">
                   <property name="text">
                    <string>Sample Images</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="paramGuiMarkButton">
                   <property name="text">
                    <string>GUI: Mark</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="paramCreateTrainingSetButton">
                   <property name="text">
                    <string>Create Training Set</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="markCheckTab">
            <attribute name="title">
             <string>check</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGraphicsView" name="checkGraphicsView"/>
             </item>
             <item>
              <widget class="QSlider" name="checkFrameSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="trainTab">
        <attribute name="title">
         <string>train</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTabWidget" name="trainTabWidget">
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>connect</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <widget class="QPushButton" name="connectConnectButton">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <widget class="QLabel" name="connectUsernameLabel">
                 <property name="text">
                  <string>username</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="connectHostnameLabel">
                 <property name="text">
                  <string>hostname</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="connectUsernameLineEdit"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="connectPasswordLabel">
                 <property name="text">
                  <string>password</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="connectHostnameLineEdit"/>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="connectPasswordLineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="processTab">
        <attribute name="title">
         <string>process</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="postprocessTab">
        <attribute name="title">
         <string>postprocess</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier 10 Pitch'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mainTabWidget</tabstop>
  <tabstop>pathsLocalLineEdit</tabstop>
  <tabstop>pathsLocalButton</tabstop>
  <tabstop>pathsNetworkVideoLineEdit</tabstop>
  <tabstop>pathsNetworkVideoButton</tabstop>
  <tabstop>pathsImportLineEdit</tabstop>
  <tabstop>pathsImportButton</tabstop>
  <tabstop>pathsImportPathsButton</tabstop>
  <tabstop>logTextEdit</tabstop>
  <tabstop>paramNameLineEdit</tabstop>
  <tabstop>paramDateLineEdit</tabstop>
  <tabstop>paramNumFrameLineEdit</tabstop>
  <tabstop>paramSelectionComboBox</tabstop>
  <tabstop>paramCroppingCheckBox</tabstop>
  <tabstop>paramSampleImagesButton</tabstop>
  <tabstop>paramGuiMarkButton</tabstop>
  <tabstop>paramCreateTrainingSetButton</tabstop>
  <tabstop>checkGraphicsView</tabstop>
  <tabstop>checkFrameSlider</tabstop>
  <tabstop>trainTabWidget</tabstop>
  <tabstop>connectHostnameLineEdit</tabstop>
  <tabstop>connectUsernameLineEdit</tabstop>
  <tabstop>connectPasswordLineEdit</tabstop>
  <tabstop>connectConnectButton</tabstop>
  <tabstop>paramCropMarginsXMin</tabstop>
  <tabstop>paramCropMarginsXMax</tabstop>
  <tabstop>paramCropMarginsYMin</tabstop>
  <tabstop>paramCropMarginsYMax</tabstop>
  <tabstop>paramMarkingsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
